### PR TITLE
Write tests

### PR DESCRIPTION
**Pull request by vladis84**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

**Comment by s1lver** [comment]
Thanks! Still need to fix Style CI

File: tests/Unit/Command/ListenCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+use Yiisoft\Yii\Queue\QueueFactoryInterface;
+use Yiisoft\Yii\Queue\QueueInterface;
+
+final class ListenCommandTest extends TestCase
+{
+    public function testConfigure(): void
+    {
+        $command = new ListenCommand($this->createMock(QueueFactoryInterface::class));
+        $chanelArgument = $command->getNativeDefinition()->getArgument('channel');
+        $this->assertEquals('channel', $chanelArgument->getName());
```
Comment (vjik) [inline tests/Unit/Command/ListenCommandTest.php:21]:
```suggestion
        $channelArgument = $command->getNativeDefinition()->getArgument('channel');
        $this->assertEquals('channel', $channelArgument->getName());
```

File: tests/Unit/Command/RunCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+use Yiisoft\Yii\Queue\QueueFactoryInterface;
+use Yiisoft\Yii\Queue\QueueInterface;
+
+final class RunCommandTest extends TestCase
+{
+    public function testConfigure(): void
+    {
+        $command = new RunCommand($this->createMock(QueueFactoryInterface::class));
+        $chanelArgument = $command->getNativeDefinition()->getArgument('channel');
+        $this->assertEquals('channel', $chanelArgument->getName());
```
Comment (vjik) [inline tests/Unit/Command/RunCommandTest.php:21]:
```suggestion
        $channelArgument = $command->getNativeDefinition()->getArgument('channel');
        $this->assertEquals('channel', $channelArgument->getName());
```

File: tests/Unit/SynchronousAdapterTest.php [hunk 3]
```diff
@@ -72,4 +72,18 @@ public function testWithAnotherChannel(): void
 
         self::assertTrue($executed);
     }
+
+    public function testStatusIdLessZero(): void
+    {
+        $adapter = $this->getAdapter();
+        $this->expectException(\InvalidArgumentException::class);
```
Comment (vjik) [inline tests/Unit/SynchronousAdapterTest.php:79]:
```suggestion
        $this->expectException(\InvalidArgumentException::class);
        $this->expectExceptionMessage('This adapter IDs start with 0.');
```

File: tests/Unit/SynchronousAdapterTest.php [hunk 4]
```diff
@@ -72,4 +72,18 @@ public function testWithAnotherChannel(): void
...
+        $adapter->status('-1');
+    }
+
+    public function testStatusNotMessage(): void
+    {
+        $adapter = $this->getAdapter();
+        $this->expectException(\InvalidArgumentException::class);
```
Comment (vjik) [inline tests/Unit/SynchronousAdapterTest.php:86]:
```suggestion
        $this->expectException(\InvalidArgumentException::class);
        $this->expectExceptionMessage('There is no message with the given ID.');
```

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Comment by vjik** [comment]
@vladis84 Thanks!

**Pull request merged**